test/acceptance/features/steps/knative_serving.py

```diff
@@ -0,0 +1,39 @@
+import re
+from command import Command
+from openshift import Openshift
+from pyshould import should_not
+
+
+class Knative_Serving():
+
+    openshift = Openshift()
+    cmd = Command()
+
+    name = ""
+    namespace = ""
+
+    def __init__(self, name="knative-serving", namespace="knative-serving"):
+        self.name = name
+        self.namespace = namespace
+        self.knative_serving_yaml_template = '''
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: {name}
+  namespace: {namespace}
+'''
+
+    def is_present(self):
+        cmd = f'oc get ks {self.name} -n {self.namespace}'
+        output, exit_status = self.cmd.run(cmd)
+        if exit_status != 0:
```

Comment:
Will printing output in case of failure helps?

Reply:
@shvasude There are so many other places too where failure output could be printed, could you take a look there as well.

Reply:
`        if exit_status != 0:
            print(f"cmd-{cmd} issued result is {output} with the exit status {exit_status}")
            return False
        return True`

Reply:
added this, let me know if this is fine

Reply:
It looks good.

Reply:
I don't think we need any statement to point out the output, I think it will be good if we can trim down the sentence to something like, `print(f"cmd: {cmd}\nExit Code: {exit_code}\nResult: {patch_output}")`.

Reply:
I think, it is okay to have some statement as well. we can quickly get into this while debugging.
also {patch_output} is part of https://github.com/redhat-developer/service-binding-operator/pull/567#discussion_r464808372

Reply:
Okay keep it but it should not unnecessarily prolix the log.

Reply:
anyways introducing logging will handle as this might go as part of debug messages

Reply:
Okay, you can resolve this conversation.